types/paypal-checkout-components/test/web.ts

```diff
@@ -0,0 +1,36 @@
+import * as paypal from "paypal-checkout-components";
```

Comment:
Does this test really work because I've been having issues since this was merged

Reply:
What issues are you having?